Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Settings/GetIshSetting.Tests.ps1

```diff
@@ -92,8 +94,10 @@ Describe “Get-IshSetting" -Tags "Read" {
 			$ishFields = Get-IshSetting -IshSession $ishSession -RequestedMetadata $requestedMetadata
 			$ishSession.DefaultRequestedMetadata = $oldDefaultRequestedMetadata
 			$ishFields.GetType().Name | Should BeExactly "Object[]"
-			if ((Get-IshVersion).MajorVersion -le 13) { $ishFields.Length | Should Be 31 }
-			if ((Get-IshVersion).MajorVersion -ge 14) { $ishFields.Length | Should Be 34 } # 14.0.0, fields FISHCOLLECTIVESPACESCFG, FISHPREVIEWRESOLUTION(value), FISHPREVIEWRESOLUTION(element) were added
+			$ishVersion = Get-IshVersion
+			if ($ishVersion.MajorVersion -le 13) { $ishFields.Length | Should Be 31 }
+			if ($ishVersion.MajorVersion -ge 14 -and $ishVersion.RevisionVersion -eq 0) { $ishFields.Length | Should Be 34 } # 14.0.0, fields FISHCOLLECTIVESPACESCFG, FISHPREVIEWRESOLUTION(value), FISHPREVIEWRESOLUTION(element) were added
```

Comment:
This test will fail on 15.0.0 as most likely 15.0.0 contains the same amount of fields as 14.0.1 (for which you created a separate test)

Reply:
Added fix.